### ISSUE TITLE
API Call to Insert Song Results in Temporary Wrong MQTT Status

### ISSUE THREAD

**Issue by MrPaulAR** [FalconChristmas/fpp#930]
**FPP Version and Hardware**: 
FPP 4.6 (OS 4.3) running on Pi 3.  It's also been occurring as far back as 4.4.

**Describe the bug and Steps to reproduce**: 
Start a playlist (scheduele or manual)
Run a POST to http://{PI}/api/command with the following body (immediate has same issue).
```
{
  "command": "Insert Playlist After Current",
  "args": [
    "Juice WRLD - Come and Go (FREE).fseq",
    "-1",
    "-1",
    "false"
  ]
}
```
When the song is over and the playlist resumes the MQTT status for FPP changes from "playing" to "idle" for the first song as the playlist is resumed.  On the 2nd song it does change back to playing.  

**Expected behavior**: 
The MQTT status should remain at playing.

**Additional context**: 
None

**Additional Attachments**
Logs are attached but I did not run this command last night. May need to grep the apache logs.
[fppd_logs.zip](https://github.com/FalconChristmas/fpp/files/5703204/fppd_logs.zip)

**Comment by cpinkham** [comment]
I see where this is in the code in Playlist::SetIdle(). We need to not send the idle status when we have a child playlist that is ending since the parent will be resumed.

**Comment by cpinkham** [comment]
This fix in master was also merged back to v4.6.
